src/everest/bin/everest_script.py

```diff
@@ -188,14 +189,19 @@ async def run_everest(options: argparse.Namespace) -> None:
         if options.gui:
             from everest.gui.main import run_gui  # noqa
 
+            client = EverestClient(
+                output_dir=options.config.output_dir,
+                config_file=options.config.config_path,
```

Comment:
This path is used to reconstruct the everest config object in `client.create_run_model_api` is there a better way of doing this?

Reply:
I guess we could extract the stuff we need from options.config and add that to the EverestClient(). Would that be better?